orga/tests/acceptance/login-test.js

```diff
@@ -106,4 +106,47 @@ module('Acceptance | Login', function (hooks) {
       assert.dom(errorMessage).exists();
     });
   });
+
+  module('when the Pix account does not exist or the password is incorrect', function () {
+    test('the login form displays a missing or invalid credentials error message', async function (assert) {
+      // given
+      domainService.getExtension.returns('fr');
+      const user = createUserWithMembershipAndTermsOfServiceAccepted();
+      createPrescriberByUser({ user });
+
+      const screen = await visit('/connexion');
+
+      // when
+      await fillByLabel(t('pages.login-form.email.label'), user.email);
+      await fillByLabel(t('pages.login-form.password'), 'incorrect-password');
+      await clickByName(t('pages.login-form.login'));
+
+      // then
+      assert.ok(screen.getByText(t('common.api-error-messages.login-unauthorized-error')));
+    });
+  });
+
+  module('when there is an unexpected error', function () {
+    test('the login form displays a generic error message', async function (assert) {
+      // given
+      domainService.getExtension.returns('fr');
+      const user = createUserWithMembershipAndTermsOfServiceAccepted();
+      createPrescriberByUser({ user });
+
+      this.server.post('/token', () => {
+        return new Response(500);
+      });
+
+      const screen = await visit('/connexion');
+
+      // when
+      await fillByLabel(t('pages.login-form.email.label'), user.email);
+      await fillByLabel(t('pages.login-form.password'), 'incorrect-password');
+      await clickByName(t('pages.login-form.login'));
+
+      // Translation common.api-error-messages.login-unexpected-error contains HTML
+      // assert.ok(screen.getByText(t('common.api-error-messages.login-unexpected-error')));
+      assert.ok(screen.getByText(new RegExp('Impossible de se connecter.')));
```

Comment:
Oubli sur l'assertion commentée ? Et comme dit plus haut : 
```suggestion:
assert.dom(screen.getByText(new RegExp('Impossible de se connecter.'))).exists();
```

Reply:
Ah oui, zut, bien vu ! Je corrige.

Reply:
✅ Assertion corrigée en utilisant `assert.dom`.